common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -1009,7 +1009,8 @@ public static UTF8String concatWs(UTF8String separator, UTF8String... inputs) {
 
     // Allocate a new byte array, and copy the inputs one by one into it.
     // The size of the new array is the size of all inputs, plus the separators.
-    final byte[] result = new byte[numInputBytes + (numInputs - 1) * separator.numBytes];
+    int intNumInputBytes = Ints.checkedCast(numInputBytes + (numInputs - 1) * separator.numBytes);
```

Comment:
Why did you use guava here instead of just checking `numInputBytes` > `Int.MaxValue`?

Reply:
this code just follow the `concat` that do the same check.
https://github.com/apache/spark/blob/c06758834e6192b1888b4a885c612a151588b390/common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java#L974

Reply:
Could you use `Math.toIntExact`, and replace `Ints.checkedCast` by `toIntExact`, please. I don't see any benefits of the Guava function over the standard function, do you?

Reply:
Replaced it.